posts/2022/10-11-graphql_subscription_setup.md

```diff
@@ -0,0 +1,246 @@
+==title==
+GrapQL subscriptions with Absinthe - the set-up
```

Comment:
```suggestion
GraphQL subscriptions with Absinthe - the setup
```

Reply:
> This is really useful.
> 
> Also, do you know that you called the PR "random test" ?

That is very embarrassing and it is called so no longer